cmd/fuzzcrypto/main.go

```diff
@@ -32,6 +32,7 @@ nor the fuzz seeding time.
 `
 
 func main() {
+	var verbose = flag.Bool("v", false, "verbose logging")
 	var fuzztime = flag.Duration("fuzztime", 5*time.Minute, "total fuzzing time")
 	var run = flagRegex("run", "regex matching a set of fuzz targets")
```

Comment:
I forgot to ask, the plan was to have it so that the fuzz tests are built in public CI but only run in internal CI. I assume this would need a new build-only flag here?

(For the first iteration I think it's fine to leave out the public CI side, because it's not super important that it's stable until the work is finished and we stop looking at the new pipeline as often.)

Reply:
The plan is to run `go build ./...` in public CI to spot compilation errors before running tests. There wouldn't be any actual test execution there, so there is no need to have a build-only flag.

Reply:
`go build ./...` is good for the tool, but it ignores `*_test` files, right?

But I suppose `go test ./...` in `cmd/fuzzcrypto` would try to build `std` and catch issues (and still not run fuzz tests because this mode only cares about "normal" tests). There's still the question of `go-cose` though... since it's in its own Go module dir, seems like it'd need another build/test command.

Reply:
> go build ./... is good for the tool, but it ignores *_test files, right?

Good catch. Check this new commit: 817fb7c598d765ba5c6201bde19ae3649327077d. I think it will solve the we-need-to-run-fuzz-as-if-they-were-tests problem. You can now pass the number of iterations to `fuzztime` (it still supports a duration) by suffixing the value with `x`, so running `go run . -fuzztime 1x` will effectively run each target one time, even `go-cose` targets. Additionally, there is no way we found a security bug with `1x`, as we will only be running the first item of the seed corpus.